packages/core/src/internal/base/base.element.scss

```diff
@@ -32,6 +32,22 @@ slot {
   display: contents !important;
 }
 
+:host([role='button']) {
+  cursor: pointer !important;
+
+  ::slotted(*) {
+    cursor: pointer !important;
+  }
+}
+
+:host([role='button'][disabled]) {
+  cursor: not-allowed !important;
+
+  ::slotted(*) {
+    cursor: not-allowed !important;
+  }
+}
+
```

Comment:
This is great. Thank you. I had added something specific in accordion for this but I believe I can remove that because this should handle it already. 👍